src/ResourceManager/ManagementPartner/Commands.Partner.Test/ScenarioTests/ManagementPartnerTests.cs

```diff
@@ -31,35 +31,35 @@ public ManagementPartnerTests(Xunit.Abstractions.ITestOutputHelper output)
             XunitTracingInterceptor.AddToContext(_logger);
         }
 
-        [Fact]
+        [Fact(Skip = "ManagementPartner tests need to be re-enabled, as outlined in issue https://github.com/Azure/azure-powershell/issues/6678")]
```

Comment:
??

Reply:
Similar to MachineLearning, these tests weren't being run during the build and I opened an issue on the ManagementPartner team to take a look. It appears that these tests have been fixed with https://github.com/Azure/azure-powershell/pull/6817, so I will update the PR to reflect this